pkg/occlient/occlient.go

```diff
@@ -1578,46 +1484,175 @@ func (c *Client) GetOnePodFromSelector(selector string) (*corev1.Pod, error) {
 	return &pods.Items[0], nil
 }
 
-// RsyncPath copies local directory to directory in running Pod.
-func (c *Client) RsyncPath(localPath string, targetPodName string, targetPath string) (string, error) {
-	log.Debugf("Syncing %s to pod %s:%s", localPath, targetPodName, targetPath)
-
-	// TODO: do this without using 'oc' binary
-	args := []string{
-		"rsync",
-		localPath,
-		fmt.Sprintf("%s:%s", targetPodName, targetPath),
-		"--exclude", ".git",
-		"--no-perms",
+// CopyFile copies single local file to the directory in running Pod.
```

Comment:
is that really true, it can just copy a single file? What if `localFile` is a directory?

Reply:
no, fixed :|